airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.java

```diff
@@ -186,11 +186,14 @@ protected void setup(final TestDestinationEnv testEnv, final HashSet<String> TES
 
   @Override
   protected void tearDown(final TestDestinationEnv testEnv, final HashSet<String> TEST_SCHEMAS) throws Exception {
```

Comment:
unrelated nit - isn't it weird that this parameter is uppercased?

Reply:
... I didn't even notice that. assumed we were using the constant directly. lemme fix it

Reply:
that diff is awful, but it's purely automated intellij refactor to remove the TEST_SCHEMAS parameter. CI is going to be dumb now but at least the first run should be snowflake alone, and I'll approve-and-merge afterwards.